src/main/resources/com/google/api/codegen/java/test.snip

```diff
@@ -80,6 +80,24 @@
         actualRequest.{@assert.actualValueGetter}());
     @end
   }
+
+  @@Test
+  @@SuppressWarnings("all")
+  public void {@test.name}Exception() {
+    {@setException(test)}
+
+    try {
+      {@initCode(test.initCode)}
+
+      api.{@test.surfaceMethodName}(\
+          {@sampleMethodCallArgList(test.initCode.fieldSettings)});
+      Assert.fail("No exception raised");
+    } catch (ApiException e) {
+      Assert.assertEquals(Status.INTERNAL.getCode(), e.getStatusCode());
+    } catch (Exception e) {
+      Assert.fail("Unknown exception found");
```

Comment:
We could remove this final catch, and just have the test declare `throws Exception`.